tests/integration/test_python_edit/test_python_edit.py

```
@@ -23,10 +23,14 @@ def setup_test_file(tmp_path, content):
         return test_file
 
 
+# Mark all tests in this file as integration tests
+pytestmark = pytest.mark.integration
+
+
 @pytest.fixture
 def test_file(tmp_path):
     """Create a test file with various Python constructs"""
-    content = "\n        import os\n        from typing import List, Optional\n\n        def outer_func():\n            def inner_func():\n                pass\n            return inner_func\n\n        class OuterClass:\n            def method(self):\n                pass\n\n            class InnerClass:\n                @staticmethod\n                def static_method():\n                    pass\n\n                async def async_method(self):\n                    pass\n\n            def another_method(self):\n                # Insert here\n                pass\n    "
+    content = "\n        import os\n        from typing import List, Optional\n\n        def outer_func():\n            def inner_func():\n                pass\n            return inner_func\n\n        class OuterClass:\n            def method(self):\n                pass\n\n            class InnerClass:\n                @staticmethod\n                def static_method():\n                    pass\n\n                async def async_method(self):\n                    pass\n\n            def another_method(self):\n                # Insert here\n                pass\n    "  # noqa: E501
```

Ruff (0.13.3)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)